### DISCUSSION TITLE
V3.5.1中的文件夹加密的密码变成了注册用户的密码？

### DISCUSSION THREAD

**Discussion by iemobiu** [AlistGo/alist#2567] [General]
如题。
在元信息里设置了一个文件夹加密，密码也设置好了。但提示输入密码的时候，提示密码错误，然后输入管理员密码就可以正常进入文件夹。
并且在输入密码下方还问是否需要注册一个帐户，还有一个登录链接。这改动的有些大了吧。
<img width="616" alt="截屏2022-12-03 20 49 36" src="https://user-images.githubusercontent.com/115678292/205441739-48673a7e-9f5c-4b0e-8aff-fad76ef9d9ca.png">

**Comment by zerofo** [comment]
> 如题。 在元信息里设置了一个文件夹加密，密码也设置好了。但提示输入密码的时候，提示密码错误，然后输入管理员密码就可以正常进入文件夹。 并且在输入密码下方还问是否需要注册一个帐户，还有一个登录链接。这改动的有些大了吧。 <img alt="截屏2022-12-03 20 49 36" width="616" src="https://user-images.githubusercontent.com/115678292/205441739-48673a7e-9f5c-4b0e-8aff-fad76ef9d9ca.png">

我有两账号， 输入管理员的密码不行， 另一个也不行。 元信息的也不行。   啥密码都不对。。。
